crates/data-plane-controller/tests/util.rs

```diff
@@ -124,7 +126,7 @@ pub fn initial_state() -> stack::State {
 
     stack::State {
         data_plane_id: models::Id::new([32; 8]),
-        deploy_branch: "main".to_string(),
+        deploy_branch: "test-branch".to_string(),
```

Comment:
Changed this so we can distinguish between git operations on infra repo from ops repo in snapshots